contracts/contracts/strategies/AaveStrategy.sol

```diff
@@ -170,19 +213,71 @@ contract AaveStrategy is InitializableAbstractStrategy {
     }
 
     /**
-     * @dev Get the current address of the Aave lending pool core, which stores all the
-     *      reserve tokens in its vault.
-     * @return Current lending pool core address
+     * @dev Collect stkAave, convert it to AAVE send to Vault.
      */
-    function _getLendingPoolCore() internal view returns (address payable) {
-        address payable lendingPoolCore = ILendingPoolAddressesProvider(
-            platformAddress
-        )
-            .getLendingPoolCore();
-        require(
-            lendingPoolCore != address(uint160(address(0))),
-            "Lending pool core does not exist"
-        );
-        return lendingPoolCore;
+    function collectRewardToken() external onlyVault nonReentrant {
+        if (address(stkAave) == address(0)) {
+            return;
+        }
+
+        // Check staked AAVE cooldown timer
+        uint256 cooldown = stkAave.stakersCooldowns(address(this));
+        uint256 windowStart = cooldown + stkAave.COOLDOWN_SECONDS();
+        uint256 windowEnd = windowStart + stkAave.UNSTAKE_WINDOW();
+        uint256 currentTimestamp = now;
```

Comment:
I prototyped this in python, then converted it to solidity, so I had this as a var since it was an external RPC call at the time.

Turns out "now" is only 2 gas, so I've just replaced currentTimestamp with it through the code.

Thanks!